### DISCUSSION TITLE
Cannot unstar on mobile if repo is listed on 'star list'

### DISCUSSION THREAD

**Discussion by sjeon87** [community/community#68265] [Repositories]
### Select Topic Area

Bug

### Body

How to reproduce:

- star a repo
- add the starred repo to a list
- visit the starred repo on mobile
- unstar the repo
- refresh the page and see the repo is still starred

(Possible) Cause

- when you unstar a repo on desktop, confirm dialog pops up if the repo is listed on a 'star list'. And after you confirm the dialog, then the repo gets unstarred
- on mobile, however, the same confirm dialog does not pop up and you cannot confirm anything. So the repo seems as if it is unstarred on frontend, however, on backend, it is not true

**Comment by alcere** [comment]
Hi @somidad, we are sorry for the troubles! Can you share with us the app version and device you are using? Thanks!

**Comment by sjeon87** [comment]
Hello, @alcere , I meant mobile version of web page

**Reply by Primus66** [reply]
Hi @somidad 
- Update Your GitHub App: If you're using the GitHub mobile app, make sure it's up to date. Check your app store for any available updates and install them if necessary.
- Check for Known GitHub Issues: GitHub actively maintains a list of known issues on their platform. You can visit the GitHub Status page (https://www.githubstatus.com/) to check if there are any ongoing issues related to starring repositories or using the mobile version.

**Reply by sjeon87** [reply]
It is responsive web page issue. It is not neither app nor server status issue

**Reply by Primus66** [reply]
I see, you're referring to an issue with the mobile version of a web page. In your case, it's related to starring and un-starring a repository on GitHub's mobile website, where the un-starring action doesn't seem to work as expected.

If you're encountering this specific issue, you can follow these steps to address it:

1. **Browser Compatibility**: Ensure that you are using a modern and up-to-date mobile browser. Older or less common browsers might not fully support certain web features, causing issues like this.

2. **Clear Browser Cache**: Sometimes, cached data in your mobile browser can cause problems. Try clearing your browser's cache and cookies, then refresh the page and attempt to un-star the repository again.

3. **Check JavaScript Errors**: Open the developer console in your mobile browser (usually accessible through browser settings or by tapping "Inspect" on certain browsers) and check for any JavaScript errors that might be related to the un-starring functionality.

4. **Try Different Mobile Browsers**: Test the issue in different mobile browsers to see if it's specific to one particular browser. This can help identify if the problem is browser-specific.

5. **Test on Different Devices**: If possible, try accessing the mobile version of the web page from different mobile devices to see if the issue is consistent across various devices.

6. **Browser Extensions**: If you have browser extensions or add-ons installed on your mobile browser, try disabling them to see if they are causing any interference.
